### DISCUSSION TITLE
How about interfacing to hrdlog.net?

### DISCUSSION THREAD

**Discussion by dg9vh** [magicbug/Cloudlog#1210] [Ideas / Feature Requests]
Hi,

I asked Claudio - IW1QLH if there is any API-documentation for his hrdlog.net - website. He was very friendly and sent me a PDF-file (23 pages) with description of several use-cases and parameters, so I think it will be easy to have calls to his website if logging a QSO or having the QSL-statuses updated (for example based on a specific start date).

If one will take the challenge, I can forward the file as confidential material to have a look at it. 

The services are some kind of XML-interface, so I think it will be very structured and straight forward to implement.

So just get in contact with me :-) Email see qrz.com
73 de Kim
DG9VH

**Comment by AndreasK79** [comment]
I've added it to the todo-list.

**Comment by AndreasK79** [comment]
@dg9vh integration is now available in the latest dev branch. Thanks to @int2001 for adding this.

**Reply by R1BLH** [reply]
Is it possible to make the field for entering the default HRDlog upload code to the user account, like LotW/eQSL credentials? 
Currently it is necessary to enter the HRDlog upload code in the profile of each station location for both newly created and existing stations. With a large number of stations this is inconvenient for the user.

**Reply by int2001** [reply]
you are able to copy the station-location. All contents will be prefilled then for the new location. including HRDLog-Credentials.

There are Hams who use different HRDLog-Accounts for different Calls/Locations. So i think it's more flexible. And there's a solution by copying it.

**Reply by magicbug** [reply]
Yep agreed best kept like it is

**Comment by do4bmw** [comment]
i dont get a eMail from HRDlog with the API code. can someone confirm that you get a new API via eMail ?

**Reply by magicbug** [reply]
Probably best to ask HRDLog directly

**Comment by dg9vh** [comment]
I got mine via ham365.net that was the same as issued via email by opening a ticket.

**Reply by do4bmw** [reply]
mny thanks , that works !
